Code/Tools/LuaIDE/Source/LUA/LUAEditorView.cpp

```diff
@@ -525,15 +526,17 @@ namespace LUAEditor
 
     void LUAViewWidget::modificationChanged(bool m)
     {
-        (void)m;
+        EBUS_EVENT(Context_DocumentManagement::Bus, NotifyDocumentModified, m_Info.m_assetId, m);
```

Comment:
Nit: Since you're moving this, could you update it to the new format?
```suggestion
        Context_DocumentManagement::Bus::Broadcast(&Context_DocumentManagement::NotifyDocumentModified, m_Info.m_assetId, m);
```

Reply:
+1